hibp.js

```diff
@@ -78,6 +82,141 @@ const HIBP = {
     } catch (error) {
       console.error(error);
     }
+    if (includeUnsafe) {
+      return foundBreaches;
+    }
+    return this.filterOutUnsafeBreaches(foundBreaches);
+  },
+
+
+  filterOutUnsafeBreaches(breaches) {
+    return breaches.filter(
+      breach => breach.IsVerified && 
+                !breach.IsRetired && 
+                !breach.IsSensitive &&
+                !breach.IsSpamList
+    );
+  },
+
+
+  getLatestBreachDateTime(breaches) {
+    let latestBreachDateTime = new Date(0);
+    for (const breach of breaches) {
+      const breachAddedDate = new Date(breach.AddedDate);
+      if (breachAddedDate > latestBreachDateTime) {
+        latestBreachDateTime = breachAddedDate;
+      }
+    }
+    return latestBreachDateTime;
+  },
+
+
+  async subscribeHash(sha1) {
+    const sha1Prefix = sha1.slice(0, 6).toUpperCase();
+    const path = "/range/subscribe";
+    const options = {
+      method: "POST",
+      body: {hashPrefix: sha1Prefix},
+    };
+
+    let response;
+    try {
+      response = await this.kAnonReq(path, options);
+    } catch (error) {
+      console.error(error);
+    }
+    return response;
+  },
+};
+
+module.exports = HIBP;
+"use strict";
```

Comment:
Something funny going on...
It almost looks like the file got duplicated from this point on.

Reply:
This seems to be what's making `eslint` fail too.